### ISSUE TITLE
Build website automatically regularly

### ISSUE THREAD

**Issue by KristinaRiemer** [terraref/tutorials#173]
To make sure website still builds correctly and code works. Maybe 1x/week? 

Probably use Travis Cron jobs: https://docs.travis-ci.com/user/cron-jobs/

**Comment by Chris-Schnaufer** [comment]
@KristinaRiemer : is this completed?

**Comment by KristinaRiemer** [comment]
No, I haven't started it beyond researching Travis jobs a bit. This definitely needs to be done, but I'm not sure where in the priority list it will go for this quarter for me.
